### ISSUE TITLE
created html/css image, font links are broken.

### ISSUE THREAD

**Issue by icq4ever** [openframeworks/ofBook#310]
for example, cannot find proper webFont "FiraMono", which is determined by `style.css` file.
while i'm investigate this issue, and image src links are broken as well.

for example, in generated html file, images are rendered as below screenshot(and check html file and actual file directory, additional screenshot.

![image](https://user-images.githubusercontent.com/530796/57390740-fe5ed780-71f7-11e9-882f-a2c12f9ab715.png)

![image](https://user-images.githubusercontent.com/530796/57390778-10d91100-71f8-11e9-9800-3b9f226ca6e3.png)

as you can see, image src is `images/laser_tag_result.jpg` in HTML, which is wrong path. it should be `../images/image_processing_computer_vision/images/laser_tag_result.jpg`.

it's not because of translated (korean) md file. other original chapter md files are also have same issue.

**Comment by icq4ever** [comment]
oh and i forgot to mention about my environment.
manjaro distro (based on arch linux).

**Comment by roymacdonald** [comment]
hi. At least that image appears in the website. is this an issue with your build? Which is the build that shows on the OF site?

**Comment by icq4ever** [comment]
@roymacdonald yes it's from by build. but I cannot find same issue on OF ofbook site. as i remember it's not happen before (locally build)
